docs/ldml/tr35.html

```diff
@@ -2369,6 +2369,29 @@ <h4><a href="#Key_And_Type_Definitions_" name=
           <em>Unknown or Invalid Currency</em>.</p>
         </td>
       </tr>
+      <tr class="changed">
+        <td colspan="4"><strong>A <a href=
+        "#UnicodeDictionaryBreakExclusionIdentifier" name=
+        "UnicodeDictionaryBreakExclusionIdentifier" id=
+        "UnicodeDictionaryBreakExclusionIdentifier">Unicode Dictionary Break Exclusion Identifier</a>
+        specifies scripts to be excluded from dictionary-based text break (for words and lines).
+        The valid values are of one or more items of type SCRIPT_CODE as specified in the
+        <em>name</em> attribute value in the <em>type</em>
+        element of key name="dx" in bcp47/<a target="_blank" href=
+        "https://github.com/unicode-org/cldr/tree/latest/common/bcp47/segmentation.xml">segmentation.xml</a>.</strong></td>
+      </tr>
+      <tr class="changed">
+        <td>"dx"</td>
+        <td>Dictionary break script exclusions</td>
+        <td>
+          <i><code><a href="#unicode_script_subtag">unicode_script_subtag</a></code> values</i>
+        </td>
+        <td>
+          <p>One or more items of type SCRIPT_CODE, which are valid
+          <code><a href="#unicode_script_subtag">unicode_script_subtag</a></code> values.</p>
+          <p>The code Zyyy (Common) can be specified to exclude all scripts.</p>
```

Comment:
Add that in that case it needs to be the only code?

Reply:
Good point, I will update to indicate this.

Reply:
Done...